virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -431,3 +432,80 @@ def get_mirror_part_in_xml(vm, disk_target):
         job_details.append(disk_mirror.get('ready'))
         job_details.append(disk_mirror.find('type'))
     return job_details
+
+
+def create_mbxml(mb_params):
+    """
+    Create memoryBacking xml
+
+    :param mb_params: dict containing memory backing attributes
+    :return memoryBacking xml
+    """
+    mb_xml = vm_xml.VMMemBackingXML()
+    for attr_key in mb_params:
+        setattr(mb_xml, attr_key,
+                mb_params[attr_key])
+    logging.debug(mb_xml)
+    return mb_xml.copy()
+
+
+def check_in_vm(vm, target, old_parts, is_equal=False):
+    """
+    Check mount/read/write disk in VM.
+
+    :param vm: VM guest.
+    :param target: Disk dev in VM.
+    :param old_parts: old part partitions
+    :param is_equal: whether two are equals
+    :return: True if check successfully.
+    """
+    try:
+        session = vm.wait_for_login()
+        rpm_stat, out_put = session.cmd_status_output("rpm -q parted || "
+                                                      "yum install -y parted", 300)
+        if rpm_stat != 0:
+            raise exceptions.TestFail("Failed to query/install parted:\n%s", out_put)
+
+        new_parts = utils_disk.get_parts_list(session)
+        added_parts = list(set(new_parts).difference(set(old_parts)))
+        logging.info("Added parts:%s", added_parts)
+        if is_equal:
+            if len(added_parts) != 0:
+                return False
```

Comment:
Better give logging.error here as below

Reply:
I think should be like "new added parts are not equal the old one"

Reply:
@kylazhang 
add  logging.error with dumping message likes 'new added parts are not equal the old one'